common/constants.go

```diff
@@ -913,6 +913,7 @@ const (
 	ESDTFlag                                           core.EnableEpochFlag = "ESDTFlag"
 	ESDTFlagInSpecificEpochOnly                        core.EnableEpochFlag = "ESDTFlagInSpecificEpochOnly"
 	GovernanceFlag                                     core.EnableEpochFlag = "GovernanceFlag"
+	GovernanceDisableProposeFlag                       core.EnableEpochFlag = "GovernanceDisableProposeFlag"
```

Comment:
can you align this constant name with the EnableEpoch flag from config?

Reply:
done